content/1.7/back-office/shop-parameters/search/search-listing.md

```diff
@@ -0,0 +1,179 @@
+- Disable selection. Disables the selected alias, changes the status to red cross mark. If no alias is selected when pressing "disable selection" the message is shown: "You must select at least one item to perform a bulk action." If an alias is selected, and "disable selection" is pressed, the message is shown: "The status has been successfully updated." and check mark is changed to a red cross.
+- Delete selected: This button will delete the selected alias. 
+After clicking on "Delete selected", a modal with the title "Delete selection" is opened asking to confirm or cancel the action: "Are you sure you want to delete the selected item(s)?" (See issue [#14462](https://github.com/PrestaShop/PrestaShop/issues/14462)). The action can be canceled by clicking on the cross or on the cancel button.
+After clicking on "Delete" button, the selected feature values are deleted, a successful message is displayed "The selection has been successfully deleted."
+
+ ## Search
+ 
+The search button is disabled as long as there are less than 2 aliases in the list. A search can be done by aliases, search, and status (drop-down menu).
+It's possible to sort the list by aliases, search in an ascending or descending way.
+After clicking on "search", the "search" button is disabled and a "reset" button is displayed.
+If there is no results for the search done, the following message is displayed: "No records found".
+When clicking on "reset", the search is removed and all the results are displayed.
+
+  # Indexing
+ 
+This section provides information on the number of products that can be searched through the shop's search bar of the front office, and compares it to the number of products present in the catalog.
+
+Firstly, an explanation is displayed: "The "indexed" products have been analyzed by PrestaShop and will appear in the results of a front office search."
+The number of products indexed and the number of products in the shop is displayed: "Indexed products 18 / 18." (the number depends on the products which are indexed and which are in the shop).
+
+Below, the following message is displayed: "Building the product index may take a few minutes. If your server stops before the process ends, you can resume the indexing by clicking "Add missing products to the index"."
+
+Below there are 2 links that performs the following actions when clicked:
+  -"Add missing products to the index" : only the new products will be indexed. After clicking on it, the page is refreshed, and the following message is displayed: "Successful  update."
+ -Re-build the entire index - the "Re-build entire index", when clicking on it: all the products are indexed and not only the missing ones compared to the previous link
```

Comment:
You should add a space before the "-" to make a list, otherwise it's impossible to read:

<img width="1043" alt="Capture d’écran 2021-11-30 à 18 25 21" src="https://user-images.githubusercontent.com/13449658/144096916-23d5a9e6-815a-48a2-8527-abe1e458ed02.png">
```suggestion
 -Re-build the entire index - the "Re-build entire index", when clicking on it: all the products are indexed and not only the missing ones compared to the previous link
```